dashboard/src/components/AppView/AppControls/AppControls.tsx

```diff
@@ -1,12 +1,16 @@
 import * as React from "react";
 import { Redirect } from "react-router";
 
-import { hapi } from "../../shared/hapi/release";
-import ConfirmDialog from "../ConfirmDialog";
-import LoadingWrapper from "../LoadingWrapper";
+import { IChartUpdate } from "shared/types";
+import { hapi } from "../../../shared/hapi/release";
+import ConfirmDialog from "../../ConfirmDialog";
+import LoadingWrapper from "../../LoadingWrapper";
+import "./AppControls.css";
+import UpgradeButton from "./UpgradeButton";
 
 interface IAppControlsProps {
   app: hapi.release.Release;
+  update?: IChartUpdate;
```

Comment:
I'm just thinking that this could be confused with an "update" action, what about if we called this and the struct `updateInfo` and `IChartUpdateInfo`?